browsers/firefox_android.json

```diff
@@ -54,237 +54,237 @@
         "42": {
           "release_date": "2015-11-03",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/42",
+          "release_notes": "https://www.mozilla.org/firefox/android/42.0beta/releasenotes/",
           "status": "retired"
         },
         "43": {
           "release_date": "2015-12-15",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/43",
+          "release_notes": "https://www.mozilla.org/firefox/android/43.0beta/releasenotes/",
           "status": "retired"
         },
         "44": {
           "release_date": "2016-01-26",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/44",
+          "release_notes": "https://www.mozilla.org/firefox/android/44.0beta/releasenotes/",
           "status": "retired"
         },
         "45": {
           "release_date": "2016-03-08",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/45",
+          "release_notes": "https://www.mozilla.org/firefox/android/45.0beta/releasenotes/",
           "status": "retired"
         },
         "46": {
           "release_date": "2016-04-26",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/46",
+          "release_notes": "https://www.mozilla.org/firefox/android/46.0beta/releasenotes/",
           "status": "retired"
         },
         "47": {
           "release_date": "2016-06-07",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/47",
+          "release_notes": "https://www.mozilla.org/firefox/android/47.0beta/releasenotes/",
           "status": "retired"
         },
         "48": {
           "release_date": "2016-08-02",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/48",
+          "release_notes": "https://www.mozilla.org/firefox/android/48.0beta/releasenotes/",
           "status": "retired"
         },
         "49": {
           "release_date": "2016-09-20",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/49",
+          "release_notes": "https://www.mozilla.org/firefox/android/49.0beta/releasenotes/",
           "status": "retired"
         },
         "50": {
           "release_date": "2016-11-15",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/50",
+          "release_notes": "https://www.mozilla.org/firefox/android/50.0beta/releasenotes/",
           "status": "retired"
         },
         "51": {
           "release_date": "2017-01-24",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/51",
+          "release_notes": "https://www.mozilla.org/firefox/android/51.0beta/releasenotes/",
           "status": "retired"
         },
         "52": {
           "release_date": "2017-03-07",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/52",
+          "release_notes": "https://www.mozilla.org/firefox/android/52.0beta/releasenotes/",
```

Comment:
This shoudn’t point to the beta release notes, but to the [normal release notes](https://www.mozilla.org/firefox/android/52.0/releasenotes/).